IdentityCore/src/cache/accessor/MSIDAccountCredentialCache.m

```diff
@@ -69,13 +69,15 @@ - (instancetype)initWithDataSource:(id<MSIDExtendedTokenCacheDataSource>)dataSou
                                                                  context:(nullable id<MSIDRequestContext>)context
                                                                    error:(NSError * _Nullable * _Nullable)error
 {
+    MSID_LOG_WITH_CTX(MSIDLogLevelInfo, context, @"(%@) retrieving cached credentials using credential query",NSStringFromClass(self.class));
```

Comment:
nit: space before NSStringFromClass here and in other places. Will NSStringFromClass affect logging performance? This might be a heavy operation.

Reply:
Good call. best to initialize it separately into another local var and then use that instead.

Reply:
However, for MSIDCredentialCacheItem, we wouldn't need to do the same, since only one of the conditionals gets executed at a time, so we can call NSStringFromClass(self.class) directly in the log.